src/utils/nats_wrapper.py

```diff
@@ -50,16 +45,16 @@ async def reconnected_cb():
             "reconnected_cb": reconnected_cb,
             "disconnected_cb": on_disconnect,
             "servers": [self.NATS_SERVER_URL],
+            "max_reconnect_attempts": -1,
```

Comment:
so no reconnect attempts?

Reply:
I think the -1 indicates that it will try reconnecting indefinitely

Reply:
indefinitely